src/amber/cli/commands/pipelines.cr

```diff
@@ -0,0 +1,92 @@
+require "cli"
+require "shell-table"
+require "../helpers/sentry"
+
+module Amber::CLI
+  class MainCommand < ::Cli::Supercommand
+    class Pipelines < Command
+      # TODO: use hash with fixed order?
+      getter result = Hash(String?, Array(String)).new
+      property current_pipe : String?
+
+      class Options
+        bool "--no-color", desc: "Disable colored output", default: false
+        help
+      end
+
+      LABELS     = ["Pipe", "Plug"]
```

Comment:
Use `%w(Pipe Plug)` instead

Reply:
Yeah, I did it, just forgot to push last updates)